src/test/java/org/simlar/simlarserver/services/pushnotification/ApplePushNotificationMockServerTest.java

```diff
@@ -35,23 +37,25 @@ public void setup() {
         mockServer = startClientAndServer();
     }
 
-    @Test
-    public void testRequestAppleVoipPushNotification() {
+    private void createMockServerRequest(final String deviceToken, final String expiration, final HttpResponse response) {
```

Comment:
Isn't `mockServer.getLocalPort()` is the only member access preventing `createMockServerRequest` from being static?